### ISSUE TITLE
v0.25 breaks collapsible TOC?

### ISSUE THREAD

**Issue by cmadland** [rstudio/bookdown#1330]
I've upgraded to bookdown v0.25 and it doesn't seem to render the TOC with sections collapsed.

```
bookdown::gitbook:
  includes:
      in_header: includes.html
  css: style.css
  config:
    toc:
      collapse: section
      scroll_highlight: true
      before: |
        <li><a href="https://otessa.org" target="blank">OTESSA</a></li>
      after: |
        <li><a href="https://bookdown.org" target="blank">Published with bookdown</a></li>
    edit: https://github.com/otessa/2022/edit/master/%s
    download: ["epub"]
bookdown::epub_book: default
```
renders to this

<img width="465" alt="image" src="https://user-images.githubusercontent.com/12282929/163273616-1b84c91c-d831-43f7-89c1-575d59a05c8a.png">

Here is a sample of how the HTML output has changed

![image](https://user-images.githubusercontent.com/12282929/163274239-c17c6fbc-f27a-4552-94b4-5e92554391af.png)

Let me know if you need any other info.

**Comment by yihui** [comment]
The issue guide (which you seem to have deleted from [the issue template](https://github.com/rstudio/bookdown/issues/new?assignees=&labels=&template=bug-report.md&title=)) asked that you try the development version of the package when reporting bugs. Did you do that? I'm asking because this is likely to have been fixed in the dev version a few days ago. Thanks!

**Comment by cmadland** [comment]
missed that...will give it a go! Thank you.

**Comment by cmadland** [comment]
Interesting that I missed the issue template. I didn't see it at all. Turns out I opened to the bookdown repo while I was logged out of GH, hit 'New Issue', logged in, and got a blank issue to work with...no  template at all. Sorry about that.

Turns out the dev version works like a charm! Thanks for all your work!

**Comment by yihui** [comment]
No worries! Good to hear that it's no longer a problem!
